Frameworks/Foundation/NSNotification.mm

```diff
@@ -85,9 +73,13 @@ - (void)dealloc {
  @Status Stub
  @Notes
 */
-- (instancetype)initWithName:(NSString*)aName object:(id)object userInfo:(NSDictionary*)userInfo {
-    UNIMPLEMENTED();
-    return StubReturn();
+- (instancetype)initWithName:(NSString*)aName object:(id)object userInfo:(NSDictionary*)info {
+    if ((self = [super init]) != nil) {
+        notificationObj = [object retain];
+        notificationName = [aName copy];
+        userInfo = [userInfo retain];
```

Comment:
info :D

---
In reply to: [69236021](https://github.com/Microsoft/WinObjC/pull/600#discussion_r69236021) [](ancestors = 69236021)